libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -813,7 +804,7 @@ async function BallotPageContent(
   );
   // For now, just one section for candidate contests, one for ballot measures.
   // TODO support arbitrarily defined sections
-  const contests = getContests({ election, ballotStyle });
+  const contests = getOrderedContests({ election, ballotStyle });
```

Comment:
I'm realizing that whenever we define contest sections, it's going to have to interact with `getOrderedContests`. Since we aren't planning to have different contest orders across ballot styles, I wonder if we should change the API to be `getOrderedCandidates` and call it for each contest to constrain where rotation applies to as localized an area as possible.

Reply:
Changing to this for now, if it ends up being easier when I get to the consumers to also have getOrderedContests I'll add it back, but I agree doing it on a contest-level kind of makes more sense now that we aren't ordering contests